util.py

```diff
@@ -116,6 +117,11 @@ class StartTile(Tile):
 
 
 class HoleTile(Tile):
+    def __init__(self, direction=None, path=None, properties=[]):
+        if direction is None:
```

Comment:
Je zde if nutný? Může se stát díra nasměrovaná jinam? Nebude lepší rovnou inicializovat díry s Direction.N?

Reply:
`def __init__(self, direction=Direction.N, path=None, properties=[]):` mi nefungovalo, proto ten if pod tím. 
A teď jsem přišla na to proč :) Předělám to. Nefungovalo to, protože definice třídy `Direction` je v kódu až za definicí `HoleTile`...